src/views/Package/Package.tsx

```diff
@@ -1,7 +1,8 @@
 import { Box, Stack } from "@chakra-ui/react";
 import { FunctionComponent, useEffect } from "react";
 import { useParams } from "react-router-dom";
-import { createAssembly } from "../../api/package/assemblies";
+import { fetchAssembly } from "../../api/package/assembly";
+import { fetchMarkdown } from "../../api/package/docs";
```

Comment:
Non-blocking: might be worth adding an index file to `api/package` so we can shorten the imports?

```ts
import { fetchAssembly, fetchMarkdown, fetchMetadata } from "../../api/package";
```